src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
```

Comment:
The condition `days >= SHORT_TERM && days < LONG_TERM` is not necessary. Since you return in the case when `days >= LONG_TERM`, you only need to check for `days >= SHORT_TERM` here.